Added functionality to install a specific maven version

PhilMFischer opened the pull request:
- created bash script to download and install a specific maven.
- Added action to setup the newly installed maven version.

Closes #21

Commit:
Adding new action for setting up maven - (Task #21)

- Added action to set a specific maven version
- added bashs cript executing the downlaod and setup

---
Task #21: Add functionality to setup correct maven version

Commit:
Fixing default version - (Task #21)

- Set default maven version to 3.9.6

---
Task #21: Add functionality to setup correct maven version

Commit:
changed parameter name - (Task #21)

 chanegd from underscore to dash

---
Task #21: Add functionality to setup correct maven version

Commit:
Fixing permissions

Commit:
-

Commit:
-

Commit:
-

Commit:
-

Commit:
-

dellerDLR commented on bash/setup_maven.sh:
```diff
@@ -0,0 +1,70 @@
+# --------------------------------------------------------------------------------------------
+# This script tries to setup the environment variables for native librarie sucha s vtk and zmq
+# --------------------------------------------------------------------------------------------
+set +e
```
Shouldn't this be a "-" to exit the script if an error occures?

PhilMFischer replied:
Now it is set to -e.

Yes, this script should properly fail if the desired maven version cannot be installed to the system. Otherwise a whole workflow might still execute but with a random maven version which is part of the ci runner.

Commit:
Change fail behavior of script (Task #21)

This script should fail as early as possible. So that the build jobs fail, if the desired maven version was not correctly installed.
